test/inject.test.js

```diff
@@ -161,7 +161,7 @@ test('inject get request - headers', t => {
   }, res => {
     t.strictEqual('world', JSON.parse(res.payload).hello)
     t.strictEqual(res.statusCode, 200)
-    t.strictEqual(res.headers['content-length'], 59)
+    t.strictEqual(res.headers['content-length'], 69)
```

Comment:
Curious. Why?

Reply:
Actually I don't know, probably we are adding more headers to the response.
`light-my-request` is slightly different from `shot`. If you are very curious, you can check out the code :)